Makefile

```diff
@@ -215,6 +215,11 @@ MK_CFLAGS    += $(WARN_FLAGS) -Wshadow -Wstrict-prototypes -Wpointer-arith -Wmis
 				-Werror=implicit-function-declaration
 MK_CXXFLAGS  += $(WARN_FLAGS) -Wmissing-declarations -Wmissing-noreturn
 
+ifeq ($(ENABLE_WERROR),1)
```

Comment:
Is there any reason to use a different flag here than in cmake? I would prefer if this was renamed to `LLAMA_FATAL_WARNINGS`.

Reply:
Updated! Thank you for reviewing the PR :)